model/src/test/java/uk/gov/hmcts/payment/api/model/Payment2RepositoryTest.java

```diff
@@ -16,32 +16,32 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.mockito.Mockito.when;
 
-@RunWith(MockitoJUnitRunner.class)
+@ExtendWith(MockitoExtension.class)
 public class Payment2RepositoryTest {
 
     @Mock
     private Payment2Repository payment2Repository;
 
     private Date fromDate;
     private Date toDate;
-    private PaymentChannel paymentChannel;
+    private PaymentChannel telephonyChannel;
 
     @BeforeEach
     public void setUp() {
         fromDate = new Date(System.currentTimeMillis() - 1000 * 60 * 60 * 24);
         toDate = new Date();
-        paymentChannel = PaymentChannel.TELEPHONY;
+        telephonyChannel = PaymentChannel.TELEPHONY;
     }
 
     @Test
     public void testFindAllByDateCreatedBetweenAndPaymentChannel_Success() {
         Tuple mockTuple1 = new CustomTuple("Service1", "CCD123", "REF123", "FEE001", new Date(), new BigDecimal("100.00"), "Success");
         Tuple mockTuple2 = new CustomTuple("Service2", "CCD456", "REF456", "FEE002", new Date(), new BigDecimal("200.00"), "Failed");
 
-        when(payment2Repository.findAllByDateCreatedBetweenAndPaymentChannel(fromDate, toDate, paymentChannel))
+        when(payment2Repository.findAllByDateCreatedBetweenAndPaymentChannel(fromDate, toDate, telephonyChannel.getName()))
             .thenReturn(Arrays.asList(mockTuple1, mockTuple2));
 
-        List<Tuple> results = payment2Repository.findAllByDateCreatedBetweenAndPaymentChannel(fromDate, toDate, paymentChannel);
+        List<Tuple> results = payment2Repository.findAllByDateCreatedBetweenAndPaymentChannel(fromDate, toDate, telephonyChannel.getName());
```

Comment:
This is very odd test, `Payment2Repository` is mocked, return result is set on mock and then mock is called to fetch what was set. Basically I'd say the test is testing that mock is working, and not the repository (nothing to address, just wanted to point out)

Reply:
I agree these tests are pointless and could do with a ticket to address the existing tech debt.